<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1445</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Busbar_SI校准工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="2" rowspan="2" colspan="2">
     <widget class="QPushButton" name="tip">
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BUSBAR_SI交直流
校准工具_V1.5</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QPushButton" name="startOrStop">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>校准</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="QTextEdit" name="debugEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="clearBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>电能清零</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="gBoxC">
      <property name="title">
       <string>操作区</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>地址码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addr">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dcBox">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>直流</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>交流</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="readBox">
           <property name="text">
            <string>读取</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="minAEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>5.5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>----</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxAEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>5.6</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="minWEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>130</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>KW</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>----</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxWEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>133</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>KW</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QWidget" name="radio" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radBtn0">
            <property name="text">
             <string>220 v 0A:校准偏移量</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radBtn10">
            <property name="text">
             <string>220v 10A:校准电流电压</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="gBoxA">
      <property name="title">
       <string>执行板串口[9600 - RS485]</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="gBoxAWid" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="vLabel">
        <property name="text">
         <string>xxx</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>设备版本号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>04b_21</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>防雷：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lpslab">
        <property name="text">
         <string>---</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5" rowspan="3">
     <widget class="QGroupBox" name="gBoxK">
      <property name="title">
       <string>使用说明：</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextEdit" name="readMe">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;校准:&lt;/span&gt;[此操作不区分地址_区分交直流]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;a.交流&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.打开串口&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2下拉选择【交流】&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3.给标准源输出[220V-6A]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4.点击校准&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;b.直流：[先校准偏移量 再校准电流电压]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.打开串口&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2.下拉选择【直流】&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3.选择相应的校准模式[0A|10A]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4.给标准源输出[220V-0A|220V-10A]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5.点击校准&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;读取:&lt;/span&gt;[此操作区分地址_不区分交直流]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.打开串口&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2.填写相应执行板地址&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3.勾选读取【按钮会变为&amp;quot;读取&amp;quot;】&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4.点击【读取】按钮&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;电能清零:&lt;/span&gt;[此操作区分地址_区分交直流]&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.打开串口&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2.填写相应执行板地址&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;3.选择交流或直流&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;4.点击【电能清零】按钮&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QGroupBox" name="gBoxH">
      <property name="title">
       <string>显示区</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>总和</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>电流（A）</string>
          </property>
          <property name="background">
           <color>
            <red>0</red>
            <green>100</green>
            <blue>0</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电压（V)</string>
          </property>
          <property name="background">
           <color>
            <red>0</red>
            <green>100</green>
            <blue>0</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电能（KWH）</string>
          </property>
          <property name="background">
           <color>
            <red>0</red>
            <green>100</green>
            <blue>0</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>温度（°C）</string>
          </property>
          <property name="background">
           <color>
            <red>0</red>
            <green>100</green>
            <blue>0</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>功率（KW）</string>
          </property>
          <property name="background">
           <color>
            <red>0</red>
            <green>100</green>
            <blue>0</blue>
           </color>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电流min（A）</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电流max（A）</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电压min（V)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>电压max（V)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>温度min（°C）</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>温度max（°C）</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>断路器</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="ChangeLineBtn">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>改成有线</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ChangeWifiBtn">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>改成无线</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Statuslab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>改无线成功</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>gBoxA</zorder>
   <zorder>gBoxC</zorder>
   <zorder>startOrStop</zorder>
   <zorder>gBoxH</zorder>
   <zorder>debugEdit</zorder>
   <zorder>clearBtn</zorder>
   <zorder>gBoxK</zorder>
   <zorder>tip</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1445</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
